_clanky/výběr-katalogizačního-systému-pro-město-Brno.md

```diff
@@ -0,0 +1,83 @@
+
+Co dodat na závěr? Výběr vhodného katalogizačního systému je náročnou disciplínou. Častokrát je těžké si správně vybrat bez předešlých zkušeností a detailní znalosti problematiky  Někdy se stane, že si samospráva (jako například v případě Brna) napoprvé nevybere pro svoje potřeby nejvhodnější technologii. Tímto samozřejmě **netvrdíme**, že CKAN nebo jiný systém je horší než systém HUB. Při výběru je totižto nutné vzít v potaz technologická, personální nebo finanční specifika každého města a ta pak určují, jaký systém je nejvhodnější. ArcGIS HUB například **určitě není vhodným řešením** pro organizace, které nemají podstatnou část svých dat ve službách nebo geodatabázích postavených nad architekturou od ESRI.
+
+Pro Brno však bylo toto řešení nejlepší z možných, neboť funkcionality a pokročilé technologické řešení popsané výše umožňují poskytovat odborné veřejnosti otevřená data bez nutnosti vynaložení velkých finančních prostředků. Kromě Brna, jako jediného většího města v ČR, využívá v současnosti ArcGIS HUB pro publikaci open dat ještě [Agentura ochrany přírody a krajiny][AOPK]. V zahraničí je užívání ArcGIS HUBu mnohem rozšířenější - využívají jej například města jako [Sydney][SYDNEY], [Washington D.C.][WASHINGTON] nebo [Pařížská metropolitní oblast][PARIS]. 
+
+Pokud Vás zajímá problematika správy otevřených dat v kontextu města Brna ještě podrobněji, neváhejte se na Brno obrátit se svým dotazem, a to na e-mail data@brno.cz.
+
+[PRHCKN]: http://https://opendata.praha.eu/ "CKAN"
+[GOLEMIO]: https://golemio.cz/ "Golemio"
+[GEOPORTAL]: https://https://www.geoportalpraha.cz/cs/data/otevrena-data/seznam "Geoportál"
+[OSTRAVU]: https://opendata.ostrava.cz/ "Ostravu"
+[PLZEN1]: https://opendata.plzen.eu/ "Plzeň"
+[CKAN]: https://ckan.org/ "CKAN"
+[GLASGOW]: http://data.glasgow.gov.uk/dataset?q= "Glasgow"
+[KANADSKA]: https://open.canada.ca/en/open-data "kanadská vláda"
+[PLZEN2]: https://opendata.plzen.eu/ "Plzeň"
+[OLOMOUC]: https://data.olomouc.eu/ "Olomouc"
+[HRADEC]: http://opendata.mmhk.cz/ "Hradec Králové"
+[NKOD]: https://data.gov.cz/datov%C3%A9-sady?dotaz= "Národní katalog otevřených dat"
+[OFN1]: https://ofn.gov.cz/rozhran%C3%AD-katalog%C5%AF-otev%C5%99en%C3%BDch-dat/2019-04-04/ "Otevřenou formální normu"
+[ENTRYSCAPE]: https://entryscape.com/en/ "Entryscape"
+[DATAPRESS]: (https://datapress.com/ "DataPress"
+[DATAWORLD]: https://data.world/ "Data.World"
+[ARCGISHUB]: https://www.esri.com/en-us/arcgis/products/arcgis-hub/overview "ArcGIS HUB"
+[DATABRNO]: (http://data.brno.cz/ "data.brno.cz"
+[WAZE]: https://data.brno.cz/search?collection=Dataset&q=waze "data například z aplikace Waze"
+[OFN2]: https://ofn.gov.cz/rozhran%C3%AD-katalog%C5%AF-otev%C5%99en%C3%BDch-dat/2019-04-04/ "OFN rozhraní katalogů otevřených dat"
+[AMERICKEHO]: https://project-open-data.cio.gov/v1.1/schema/ "amerického"
+[OFN3]: https://ofn.gov.cz/rozhran%C3%AD-katalog%C5%AF-otev%C5%99en%C3%BDch-dat/2019-04-04/ "podoby, kterou požaduje NKOD"
```

Comment:
Tento link už je definovaný 2x - OFN1 a OFN2 . Link není třeba definovat tolikrát, kolikrát se vyskytuje. Lze použít jednu definici.

Text linku v definici je alternativní text, který má popisovat, co za tím linkem je. tedy ne "podoby, kterou požaduje NKOD", ale "Otevřená formální norma Rozhraní katalogů otevřených dat". A pak to stačí právě jednou.

Konkrétně zde bych ale dával URL bez verze na konci, aby po vydání nové verze odkaz vedl vždy na ni, a ne stále na tu první.

Reply:
upravené